src/cli/check/mod.rs

```diff
@@ -0,0 +1,450 @@
+use std::fmt::Debug;
+
+use clap::{Args, Subcommand};
+use colored::{Color, ColoredString, Colorize};
+use log::{debug, info};
+use serde_json::Value;
+
+use crate::errors::{Error, Result};
+
+use self::template::TemplateChecker;
+
+use super::CliCommand;
+
+mod template;
+
+pub const RULE_SEPARATOR: &str = ":";
+pub const WIDGET: &str = "widget";
+
+#[derive(Args, Clone)]
+pub struct Check {
+    #[clap(subcommand)]
+    command: CheckCommandType,
+}
+
+/// The check subcommands
+#[derive(Subcommand, Clone)]
+pub enum CheckCommandType {
+    // /// Check the current project as an app
+    // App(CheckParameters),
+    /// Check the current project as a template
```

Comment:
Add documentation to explain how to do the check (start the application using --expose)

Reply:
@taorepoara

Reply:
It's already documented: `lenra check --help`